packages/fxa-auth-server/lib/senders/emails/partials/subscriptionCharges/en.ftl

```diff
@@ -1,21 +1,10 @@
 # Variables:
-#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
-subscriptionFirstInvoiceDiscount-subject = { $productName } payment confirmed
-# Variables:
-#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
-subscriptionFirstInvoiceDiscount-title = Thank you for subscribing to { $productName }
-subscriptionFirstInvoiceDiscount-content-processing = Your payment is currently processing and may take up to four business days to complete.
-# Variables:
-#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
-subscriptionFirstInvoiceDiscount-content-install-2 = You will receive a separate email on how to start using { $productName }.
-subscriptionFirstInvoiceDiscount-content-auto-renew = Your subscription will automatically renew each billing period unless you choose to cancel.
-# Variables:
-#  $invoiceNumber (String) - The invoice number of the subscription invoice, e.g. 8675309
-subscriptionFirstInvoiceDiscount-content-invoice-number = Invoice Number: <b>{ $invoiceNumber }</b>
-# Variables:
 #  $invoiceNumber (String) - The invoice number of the subscription invoice, e.g. 8675309
```

Comment:
Nit: I think we can use the existing ftl IDs that don't include `Discount` (e.g `subscriptionFirstInvoice-content-invoice-number`), I feel like it'd be cleaner since `subscriptionFirstInvoiceDiscount` / `subscriptionSubsequentInvoiceDiscount` email templates will no longer be used. But again it's a nit since those are existing ftl IDs as well.